### PR TITLE
WIP: technical director berlin

### PR DESCRIPTION
**Pull request by ebuchman**
Don't merge yet

### CODE REVIEW

File: md/careers/technical-director-berlin.md [hunk 1]
```diff
@@ -1,30 +1,24 @@
...
 * Built a number of decentralized applications.
+* Experience running developer community events.
 * The ability to take ownership and see initiatives through.
 * Exceptional communication skills.
-* Experience working with distributed teams.
+* Experience leading and working with distributed teams.
 
 ### What your primary responsibilities will be:
 
+* Manage the Berlin engineering team including overseeing hiring/termination and performance management.
```
Comment (jolesbi) [inline md/careers/technical-director-berlin.md]: maybe we don't have to include termination in the job description.

File: md/careers/technical-director-berlin.md [hunk 2]
```diff
@@ -1,30 +1,24 @@
...
+* Responsible for the  planning, execution, and success of complex technical projects.
```
Comment (jolesbi) [inline md/careers/technical-director-berlin.md]: extra space?

File: md/careers/technical-director-berlin.md [hunk 3]
```diff
@@ -1,30 +1,24 @@
...
 * The ability to take ownership and see initiatives through.
 * Exceptional communication skills.
-* Experience working with distributed teams.
+* Experience leading and working with distributed teams.
 
 ### What your primary responsibilities will be:
 
+* Manage the Berlin engineering team including overseeing hiring/termination and performance management.
+* Responsible for the  planning, execution, and success of complex technical projects. 
 * Manage and oversee the Cosmos FullNode community in Berlin to support ecosystem
```
Comment (jolesbi) [inline md/careers/technical-director-berlin.md]: "projects in the cosmos ecosystem"

File: md/careers/technical-director-berlin.md [hunk 4]
```diff
@@ -1,30 +1,24 @@
...
-of projects around Cosmos
-* Support a growing community of developers in the blockchain ecosystem by hosting
+of projects around Cosmos. 
+* Support a growing community of developers in the blockchain ecosystem by overseeing
```
Comment (jolesbi) [inline md/careers/technical-director-berlin.md]: overseeing meetups or planning and hosting?

**Comment by jolesbi** [comment]
this really feels like two different roles. if we hire an engineering manager in berlin (which sounds great to me) surely they could help support our engineers across all timezones (which would be a full time job). 

managing engineers is a very different skillset than hosting events and doing event marketing.

File: md/careers/technical-director-berlin.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Engineering + Community Manager
+
+### This is a full time position and is located in Berlin
+
+### We're looking for someone who has:
+ 
+* Experience leading a team of engineers including task planning and code reviews. 
+* Experience building software.
+* Built a number of decentralized applications.
```
Comment (zmanian) [inline md/careers/technical-director-berlin.md]: Do we really need them to have built a decentralized application before?

**Comment by zramsay** [comment]
closing, leaving branch. re-open in tendermint website repo if still wanted

**Pull request closed without merging**